models/index.ts

```diff
@@ -172,14 +172,43 @@ export interface summarizeRequest {
   additional_command?: string;
 }
 
+export interface rerankRequest {
+  /**
+   * A list of document objects to rerank - it is assumed the object has a text key.
+   */
+  documents: string[] | { text: string }[];
+  /** The search query that you would like the documents to be ranked against */
+  query: string;
+  /** Denotes the model to be used:
+   * An English model, rerank-english-v2.0.
+   * A multilingual model , rerank-multilingual-v2.0.
+   */
+  model: string;
```

Comment:
```suggestion
  model?: rerankModel;
```

```
  export enum rerankModels {
    RERANK_ENGLISH_V2 = "rerank-english-v2.0",
    RERANK_MULTILINGUAL_V2 = "rerank-multilingual-v2.0",
  }
```

I think we can use a more specific type here for the model since it only accepts specific strings, and enum would be better.

Also, according to the docs, it seems that this is an optional field. Hence adding the "?"

Reply:
Thanks for the comments and sorry for not taking any action on the reviews. I had already given up at merging the PR at that point 😅